configure

```diff
@@ -1351,7 +1351,9 @@ def configure_inspector(o):
   o['variables']['v8_enable_inspector'] = 0 if disable_inspector else 1
 
 
-def get_bin_override():
+def make_bin_override():
+  if sys.platform == 'win32':
+    raise Exception('make_bin_override should not be called on win32.')
```

Comment:
Does WSL's `sys.platform` identify as `win32`?

Reply:
It says `linux2`